tensorrt_llm/_torch/models/modeling_nemotron_nano.py

```
@@ -2521,45 +2521,89 @@ def merge_evs_mm_embeds(
         # Iterate over batch, replacing video_context_token_id placeholders with
         # the actual per-tubelet img_context_token counts from EVS.
         context_parts = []
-        for evs_ids, modality, num_tokens_in_video in zip(
-            evs_ids_lst, modalities, num_tokens_in_videos
+        for multimodal_param, evs_ids, modality, num_tokens_in_video in zip(
+            multimodal_params, evs_ids_lst, modalities, num_tokens_in_videos
         ):
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)